programs/sbf/rust/custom_heap/src/lib.rs

```diff
@@ -50,7 +51,7 @@ unsafe impl std::alloc::GlobalAlloc for BumpAllocator {
         // I'm a bump allocator, I don't free
     }
 }
-#[cfg(not(test))]
+#[cfg(all(not(test), target_os = "solana"))]
```

Comment:
It might be even clearer to simplify this to: 
```suggestion
#[cfg(target_os = "solana")]
```

What do you think?

Reply:
sure! let's do this [da0cdd4](https://github.com/anza-xyz/agave/pull/1312/commits/da0cdd4473853ad6607c18fdcaeef77a095cbe1c)